src/progress-bar.tsx

```diff
@@ -21,6 +23,7 @@ const ProgressBar: React.FC<Props> = ({
     'aria-label': ariaLabel,
     'aria-labelledby': ariaLabelledBy,
     dataAttributes,
+    inverse = false,
```

Comment:
Don't use 'inverse' for this prop, this name cause conflicts between `inverse` prop in other component that shows inverse state (for color backgrounds)

alternatives...
- inverted?
- animationInverted?

@atabel wdyt?